### PR TITLE
2065 local forging support new operations for lima

### PR DESCRIPTION
**Pull request by hui-an-yang**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/data/allTestsCases.ts [hunk 1]
```diff
@@ -1410,3 +1412,83 @@ export const kathmanduCases: TestCase[] = [
     },
   },
 ];
+
+
+export const limaCases: TestCase[] = [
+  // In `opMapping` from the file `constants.ts`, the operations and types starting at `ticket` were added in the lima protocol
+  ...extractOp(154, 154, opMapping).map((op): TestCase => {
```
Comment (roxaneletourneau) [inline integration-tests/data/allTestsCases.ts:1419]: Can we test from 152 to 154 to include lambda_rec as well?

File: integration-tests/data/allTestsCases.ts [hunk 2]
```diff
@@ -1345,7 +1347,7 @@ export const commonCases: TestCase[] = [
 ];
 
 export const kathmanduCases: TestCase[] = [
-  ...extractOp(150, 151, opMapping).map((op): TestCase => {
+  ...extractOp(150, 151, opMappingProto14).map((op): TestCase => {
```
Comment (roxaneletourneau) [inline integration-tests/data/allTestsCases.ts:1350]: Can we move the following tests:
- Origination of a contract that contains the instructions EMIT
- Increase Paid Storage operation
into `commonCases` ? Those operation/instruction are now supported on every protocol we are testing against (Kathmandu, Lima, alpha and upcoming)

**Review by roxaneletourneau — COMMENTED** [review]
Good work, some minor comments about the tests. 

Can you open an issue to remind us to delete "taquito-local-forging/src/proto14-kathmandu/" when we will be working on M protocol?

File: .github/workflows/deploy_website.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
-        node-version: 16.13.1
+        node-version: 16
```
Comment (dsawali) [inline .github/workflows/deploy_website.yml:20]: what's the reasoning behind this change?

Comment (dsawali) [reply .github/workflows/deploy_website.yml:20]: is this perhaps from master?

**Comment by dsawali** [comment]
Does this branch have changes from the recent `master`?

**Pull request closed without merging**